sympy/simplify/tests/test_trigsimp.py

```diff
@@ -357,6 +358,14 @@ def test_issue_2827_trigsimp_methods():
     eq = 1/sqrt(E) + E
     assert exptrigsimp(eq) == eq
 
+def test_issue_15129_trigsimp_methods():
+    t1 = Matrix([sin(Rational(1, 50)), cos(Rational(1, 50)), 0])
+    t2 = Matrix([sin(Rational(1, 25)), cos(Rational(1, 25)), 0])
+    t3 = Matrix([cos(Rational(1, 25)), sin(Rational(1, 50)), 0])
+    r1 = t1.dot(t2)
+    r2 = t1.dot(t3)
+    assert r1.simplify() == cos(1/50)
```

Comment:
This could be `trigsimp(r1) == cos(S(1)/50)`. `simplify` is an expensive operation that tries many different methods, most of which only apply in specific cases, while we are expressly testing `trigsimp`. (This could also be in release notes: change order of TR10i and TRmorrie in trigsimp.)